aiounifi/dpi.py

```diff
@@ -71,7 +47,7 @@ def site_id(self) -> str:
         return self.raw["site_id"]
 
 
-class DPIRestrictionApps(DPIItems):
+class DPIRestrictionApps(APIItems):
```

Comment:
I believe this is the culprit for https://github.com/home-assistant/core/issues/57845

`DPIItems` processes state differently than `APIItems`.

Reply:
Thanks! I will have a look tonight!